### PR TITLE
remove IS_AMP_ALT

### PR DESCRIPTION
**Pull request by erwinmombay**
This flag was only needed for single pass which is abandoned

### CODE REVIEW

**Comment by samouri** [comment]
🔥 🔥 🔥 🔥

File: src/amp.js [hunk 1]
```diff
@@ -52,19 +52,6 @@ import {preconnectToOrigin} from './preconnect';
...
- * main v0.js since it is the "main" js.
- * This global boolean is set by alternative binaries like amp-inabox and
- * amp-shadow which has their own bootstrapping sequence.
- * With how single pass works these alternative binaries cannot be generated
- * easily because we can only do a "single pass" so we treat these alternative
- * main binaries as "extensions" and we concatenate their code with the main
- * v0.js code.
- * @type {boolean|undefined}
- */
-const shouldMainBootstrapRun = !self.IS_AMP_ALT;
```
Comment (samouri) [inline src/amp.js:66]: According to the description, don't we still need this variable for `amp-inabox` and `amp-shadow`?

**Review by kevinkimball — APPROVED** [review]
LGTM.

File: src/amp.js [hunk 2]
```diff
@@ -113,30 +100,39 @@ function bootstrap(ampdoc, perf) {
...
+  // Declare that this runtime will support a single root doc. Should happen
+  // as early as possible.
+  installDocService(self, /* isSingleDoc */ true);
+  ampdocService = Services.ampdocServiceFor(self);
+} catch (e) {
+  // In case of an error call this.
+  makeBodyVisibleRecovery(self.document);
+  throw e;
+}
+startupChunk(self.document, function initial() {
```
Comment (jridgewell) [inline src/amp.js:127]: Looks like this was duplicated?

**Review by jridgewell — CHANGES_REQUESTED** [review]

File: src/amp.js (hunk 1 above)
Comment (erwinmombay) [reply src/amp.js:66]: sorry late reply. this should have been only needed for single pass, if theres any other description i haven't removed about it, could you point it out? thanks

File: src/amp.js (hunk 2 above)
Comment (erwinmombay) [reply src/amp.js:127]: looks like bad rebase on my part. re-pushed and re-checked with w=1, seems good now. but would appreciate another look from you. thanks

**Review by jridgewell — APPROVED** [review]

**Pull request merged**